src/components/content/Content.jsx

```diff
@@ -0,0 +1,20 @@
+import CreateTasks from "src/components/tasks/CreateTasks";
+import TaskItems from "src/components/tasks/TaskItems";
+import "src/components/content/Content.component.scss";
+import { useState } from "react";
+import CreateButton from "src/components/tasks/CreateButton";
```

Comment:
bundle files are easy to understand that what are importing in the files.
```suggestion
import { useState } from "react";
import "src/components/content/Content.component.scss";
import CreateTasks from "src/components/tasks/CreateTasks";
import TaskItems from "src/components/tasks/TaskItems";
import CreateButton from "src/components/tasks/CreateButton";
```